testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/messaging/mgmt/JMSBridgeManagementTestCase.java

```diff
@@ -0,0 +1,360 @@
+        reload();
+        final ModelControllerClient client = this.managementClient.getControllerClient();
+        ModelNode operation = Util.createOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        operation.get(INCLUDE_RUNTIME).set(true);
+        operation.get(NAME).set(STARTED);
+        ModelNode result = client.execute(operation);
+        Assert.assertTrue("Bridge should start!",result.get(RESULT).asBoolean());
+        
+        operation = Util.createOperation(STOP, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        result = client.execute(operation);
+        operation = Util.createOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        operation.get(INCLUDE_RUNTIME).set(true);
+        operation.get(NAME).set(STARTED);
+        result = client.execute(operation);
+        Assert.assertFalse("Bridge should stop!",result.get(RESULT).asBoolean());
+
+        operation = Util.createOperation(START, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        result = client.execute(operation);
+        operation = Util.createOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        operation.get(INCLUDE_RUNTIME).set(true);
+        operation.get(NAME).set(STARTED);
+        result = client.execute(operation);
+        Assert.assertTrue("Bridge should start!",result.get(RESULT).asBoolean());
+    }
+
+    @Override
+    protected ModelControllerClient getModelControllerClient() {
+        return this.managementClient.getControllerClient();
+    }
+
+    public void reload() throws Exception {
+        executeReloadAndWaitForCompletion(this.managementClient.getControllerClient(), 100000);
+    }
+    
+    
+    static class SetupConnectionFactory implements ServerSetupTask {
+
+        @Override
+        public void setup(final ManagementClient managementClient, final String containerId) throws Exception {
+            final ModelNode operation = Util.createAddOperation(ADDRESS_XA_CONNECTION_FACTORY);
+            operation.get("factory-type").set("XA_GENERIC");
+            //operation.get("connector", "netty").set("netty");
+            operation.get(CommonAttributes.CONNECTORS).add("netty");
+            operation.get("entries").add("java:jboss/jms/XAConnectionFactory")
+                    .add("java:jboss/exported/jms/XAConnectionFactory");
+
+            ModelNode result = managementClient.getControllerClient().execute(operation);
+            Assert.assertTrue(result.toString(), Operations.isSuccessfulOutcome(result));
+        }
+
+        @Override
+        public void tearDown(final ManagementClient managementClient, final String containerId) throws Exception {
+            managementClient.getControllerClient().execute(Util.createRemoveOperation(ADDRESS_XA_CONNECTION_FACTORY));
+        }
+    }
+
+    class SetupQueuesFactory implements ServerSetupTask {
```

Comment:
org.jboss.as.test.integration.common.jms.JMSOperations could be used to create the queues